warning: cast to pointer from integer of different size

### Environment
- TeXstudio: <!-- VERSION 	--> all
- Qt: <!-- SEE THE ABOUT SCREEN IN TXS --> 6.11.0
- OS: <!-- Windows(7,8,10), Mac, Linux (Distribution), ... --> Win
- TeX distribution: <!-- miktex, texlive, ... --> any

### Description
<!-- add as much describing information about your issue as necessary -->
While building txs with msys2 following message is thrown:
```
/texstudio/src/debug/debughelper.cpp:322:56: warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]
  322 |     if (!(*SymInitialize)(true ? GetCurrentProcess() : (HANDLE)GetCurrentProcessId(), 0, true))
      |                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Since the ternary operator always selects the then path a simple solution could remove the message:
```
if (!(*SymInitialize)(GetCurrentProcess(), 0, true))
```
Before I proceed, I’d like to check if this change is acceptable to you.

--- a/src/debug/debughelper.cpp
+++ b/src/debug/debughelper.cpp
@@ -319,8 +319,8 @@ bool initDebugHelp()
 
 	LOAD_FUNCTIONREQRET(SymInitialize, "SymInitialize", false);
 
-    if (!(*SymInitialize)(true ? GetCurrentProcess() : (HANDLE)GetCurrentProcessId(), 0, true))
-        return false;
+	if (!(*SymInitialize)(GetCurrentProcess(), 0, true))
+		return false;
 	return true;
 }
 

